README.md

```diff
@@ -2,13 +2,35 @@
 
 Snap recipe for the [Intel NPU Driver](https://github.com/intel/linux-npu-driver/). This snap is designed to be a producer snap providing NPU (neural processing unit) firmware, char device node access, and user-space libraries (including the user mode driver and NPU compiler) for consumption by application snaps. It exposes slots for consumer snaps to connect to (see below) but also provides firmware binary blobs for the NPU device and packages an app for validating the user space driver (`vpu-umd-test`).
 
+## Host OS Support
+
+### Meteor Lake
+
+The [`vpu-umd-test` user mode driver validation tool](#running-the-vpu-umd-test-application) is used to validate the snap with the following host OS + kernel on a [Intel Core Ultra 7 155H](https://www.intel.com/content/www/us/en/products/sku/236847/intel-core-ultra-7-processor-155h-24m-cache-up-to-4-80-ghz/specifications.html).
+
+| Host OS | Kernel Version | NPU Kernel Driver Support | Test Results | Comments |
+| ----- | :--: | :----------------: | :------------: | :------------------------------: |
+| 22.04 | 5.15 | :x:                | N/A            | Standard 22.04 kernel            |
+| 22.04 | 6.8  | :white_check_mark: | 184/199 passed | Hardware enablement (HWE) kernel |
+| 24.04 | 6.8  | :white_check_mark: | 184/199 passed | Standard 24.04 kernel            |
+| 24.10 | 6.11 | :white_check_mark: | 190/199 passed | Proposed 24.10 kernel            |
```

Comment:
@frenchwr i think it is better to provide `uname -r` in addition or in place of `Standard 24.04 kernel`.

Reply:
My intention was to make it clear that the only special case is 22.04 where a HWE kernel is needed. Maybe I could go a step further and list the apt `linux-image-*` package name for each? My concern about `uname -r` is that for 22.04 that would not make it clear that this is a supported HWE kernel (it still has `-generic` in the name).

Reply:
ah I see your concern, as you want as long as it makes things clear. Thanks